From 200f71ea8c0756594ac7e079ccc686d9a20cea5c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 23 Nov 2020 15:32:58 -0800
Subject: [PATCH] backtrace: Always include stdint.h

in OE we will always have system headers which supports C99/stdint.h

Upstream-Status: Inappropriate [Unless upstream drops legacy]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 3rdparty/libbacktrace/backtrace.h | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/3rdparty/libbacktrace/backtrace.h b/3rdparty/libbacktrace/backtrace.h
index 14863cf..d0ac38f 100644
--- a/3rdparty/libbacktrace/backtrace.h
+++ b/3rdparty/libbacktrace/backtrace.h
@@ -36,24 +36,8 @@ POSSIBILITY OF SUCH DAMAGE.  */
 #include <stddef.h>
 #include <stdio.h>
 
-/* We want to get a definition for uintptr_t, but we still care about
-   systems that don't have <stdint.h>.  */
-#if defined(__GLIBC__) && __GLIBC__ >= 2
-
-#include <stdint.h>
-
-#elif defined(HAVE_STDINT_H)
-
 #include <stdint.h>
 
-#else
-
-/* Systems that don't have <stdint.h> must provide gstdint.h, e.g.,
-   from GCC_HEADER_STDINT in configure.ac.  */
-#include "gstdint.h"
-
-#endif
-
 #ifdef __cplusplus
 extern "C" {
 #endif
